README.md

```diff
@@ -20,4 +20,4 @@ npm start
 
 Launch the Development Client app on your phone and enter the URL you see on your terminal (you can scan the QR code to get the URL).
 
-[dev-client]: https://expo.dev/accounts/nop33/projects/mobile-wallet/builds/2b937393-122d-4b37-b53a-4985e69a23d3
+[dev-client]: https://expo.dev/accounts/nop33/projects/mobile-wallet/builds/00080e0a-fca8-4b43-b988-e91e861227ed
```

Comment:
Do we know how for long these dev builds are hosted?

Reply:
Good question. It appears that it's 30 days.

We could:
- Commit it to this repo
- Commit it to another repo

<img width="391" alt="image" src="https://user-images.githubusercontent.com/1579899/205596130-5c3dd7fe-9c40-483d-8929-2635f582a99b.png">

Reply:
Github is our go-to "hosting" service, so it would be great to use it for that.
Having them in this repo would make sense IMO, as they are required to actually be able to contribute to the project.